pcsx2-qt/QtUtils.cpp

```diff
@@ -129,6 +132,42 @@ namespace QtUtils
 		ResizeColumnsForView(view, widths);
 	}
 
+	void ShowInFileExplorer(QWidget* parent, const QFileInfo& file)
+	{
+#if defined(_WIN32)
+		std::wstring wstr = QDir::toNativeSeparators(file.absoluteFilePath()).toStdWString();
+		bool ok = false;
+		if (PIDLIST_ABSOLUTE pidl = ILCreateFromPath(wstr.c_str()))
+		{
+			ok = SUCCEEDED(SHOpenFolderAndSelectItems(pidl, 0, nullptr, 0));
+			ILFree(pidl);
+		}
+#elif defined(__APPLE__)
+		bool ok = CocoaTools::ShowInFinder(file.absoluteFilePath().toStdString());
+#else
+		bool ok = QDesktopServices::openUrl(QUrl::fromLocalFile(file.absolutePath()));
+#endif
+		if (!ok)
+		{
+			QMessageBox::critical(parent, QObject::tr("Failed to show file"),
+				QObject::tr("Failed to show file in file explorer.\n\nThe file was: %1").arg(file.absoluteFilePath()));
```

Comment:
tr() isn't going to work here, because it doesn't have a scope. Use `qApp->translate(context, msg)` instead.

Reply:
Somehow used that 10 lines later, but not here, amazing